compliance/cis_k8s/rules/cis_4_2_12/data.yaml

```diff
@@ -38,7 +38,7 @@ metadata:
   impact: |
     None
   default_value: |
-    By default, kubelet server certificate rotation is disabled.
+    By default, kubelet server certificate rotation is enabled.
```

Comment:
please open a PR to update that here: https://github.com/elastic/integrations/blob/8dd73d610ab2d2ffdd3df90b1b2fafd14ca37b7d/packages/cloud_security_posture/kibana/csp_rule_template/46d69440-3946-50f6-83b3-a0987551afa2.json#L13

Reply:
On it